documentation/docs/05-misc/04-migrating-v3-to-v4.md

```diff
@@ -2,4 +2,56 @@
 title: Migrating from v3 to v4
 ---
 
-TODO
+This migration guide provides an overview of how to migrate from Svelte version 3 to 4. See the linked PRs for more detailed migration instructions. Use the migration script to migrate some of these automatically: `npx svelte-migrate svelte-4`
```

Comment:
"See the linked PRs for more detailed migration instructions" makes it sound like this guide won't be thorough enough. Maybe we can reword?
```suggestion
This migration guide provides an overview of how to migrate from Svelte version 3 to 4. See the linked PRs for more details about each change. Use the migration script to migrate some of these automatically: `npx svelte-migrate svelte-4`
```